modules/installation-restricted-network-samples.adoc

```diff
@@ -105,14 +105,14 @@ This is required because the imagestream import process does not use the mirror
 ====
 +
 . Add any imagestreams that are not mirrored into the `skippedImagestreams` field
-of the Samples Operator configuration object. Or if you do not want to support
-any of the sample imagestreams, set the Samples Operator to `Removed` in the
-Samples Operator configuration object.
+of the Cluster Samples Operator configuration object. Or if you do not want to support
+any of the sample imagestreams, set the Cluster Samples Operator to `Removed` in the
+Cluster Samples Operator configuration object.
 +
 [NOTE]
 ====
-Any unmirrored imagestreams that are not skipped, or if the Samples Operator is
-not changed to `Removed`, will result in the Samples Operator reporting a
+Any unmirrored imagestreams that are not skipped, or if the Cluster Samples Operator is
+not changed to `Removed`, will result in the Cluster Samples Operator reporting a
```

Comment:
Ah shoot found another item I missed.  In 4.6 only, we backed off marking degraded with https://github.com/openshift/cluster-samples-operator/commit/ecfb9e61ee1570abccefc77fa9b085fcfb5bfbf1

And as a replacement, it issues alerts if a) imports are failing but samples operator is periodically retrying them, b) imports are failing and samples operator does not appear to be retrying them.

Reply:
Brought forward to https://github.com/openshift/openshift-docs/pull/26134/.